LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
+
+Copyright (c) 2018, Juan Manuel Martinez Caamaño and Serge Guelton
```

Comment:
I think you should also add Quarkslab in the copyright

Reply:
Done!